src/goto-programs/goto_program.cpp

```diff
@@ -19,10 +19,15 @@ Author: Daniel Kroening, kroening@kroening.com
 Function: goto_programt::output_instruction
 
   Inputs:
+   ns - the namespace to resolve the expressions in
+   identifier - the identifier used to find a symbol to identify the
+                source language
+   out - the stream to write the goto string to
+   it - an iterator pointing to the instruction to convert
 
- Outputs:
+ Outputs: See below.
 
- Purpose:
+ Purpose: See below.
```

Comment:
At some point we may have proper doxygen (AFAIK @peterschrammel has a patch to transform all comments), where proper referencing can be used.